packages/jellyfish-wallet-mnemonic/src/mnemonic.ts

```diff
@@ -33,10 +34,10 @@ export function validateMnemonicWord (word: string): boolean {
  * |  256  |  8 |   264  |  24  |
  *
  * @param {number} length the sentence length of the mnemonic code
- * @param {(number) => Buffer} rng random number generation, generate random num of bytes buffer
+ * @param {(number) => Buffer} [rng = randomBytes] cryptographically strong random values generator required
  * @return {string[]} generated mnemonic word list, (COLD STORAGE)
  */
-export function generateMnemonicWords (length: 12 | 15 | 18 | 21 | 24 = 24, rng?: (numOfBytes: number) => Buffer): string[] {
+export function generateMnemonicWords (length: 12 | 15 | 18 | 21 | 24 = 24, rng: (numOfBytes: number) => Buffer = randomBytes): string[] {
```

Comment:
hmm... wondering isn't it using `crypto.randomBytes` safer than `randomBytes`?

```suggestion
export function generateMnemonicWords (length: 12 | 15 | 18 | 21 | 24 = 24, rng: (numOfBytes: number) => Buffer = crypto.randomBytes): string[] {
```

Reply:
ignore above.. same thing..

Reply:
It's all about platform agnostic (compatibility issue)

- crypto package not available on mobile (at least not a drop in replacement easily)
- randombytes (this package is browserify but still not working for mobile)
- last resort (client dev override with whatever work for them)